practice/c/swap.c

```
@@ -0,0 +1,20 @@
+#include<stdio.h>
+
+// using pointers, swap two numbers
+
+int main() {
+    int a, b;
```

Clang (14.0.6)

[warning] multiple declarations in a single statement reduces readability

(readability-isolate-declaration)